### PR TITLE
new post: deploying quarkus to openshift using intellij

### PR DESCRIPTION
**Pull request by adietish**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

**Comment by adietish** [comment]
@maxandersen here you are. Looking forward to your comments.

**Comment by adietish** [comment]
@maxandersen Any updates, please? Would be great if we could have this moving forward?

File: _posts/2023-09-11-live-diff-and-update-using-idea.adoc [hunk 1]
```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: Live diff and update quarkus deployments in OpenShift using Jetbrains IDEA
+date: 2023-09-11
```
Comment (maxandersen) [inline _posts/2023-09-11-live-diff-and-update-using-idea.adoc:4]:
```suggestion
date: 2023-09-29
```

**Review by maxandersen — APPROVED** [review]

File: _posts/2023-09-11-live-diff-and-update-using-idea.adoc [hunk 2]
```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: Live diff and update quarkus deployments in OpenShift using Jetbrains IDEA
+date: 2023-09-29
```
Comment (maxandersen) [inline _posts/2023-09-11-live-diff-and-update-using-idea.adoc:4]:
```suggestion
date: 2023-09-28
```

**Review by maxandersen — APPROVED** [review]

**Comment by maxandersen** [comment]
looks good -sorry for delay due to travels.

**Comment by adietish** [comment]
@maxandersen wohoo, awesomeness, thanks!

**Pull request merged**